gordo_components/dataset/datasets.py

```diff
@@ -82,14 +93,31 @@ def get_data(self) -> Tuple[pd.DataFrame, Optional[pd.DataFrame]]:
             tag_list=list(set(self.tag_list + self.target_tag_list)),
         )
         data: pd.DataFrame = self.join_timeseries(
-            series_iter, self.from_ts, self.to_ts, self.resolution
+            series_iter,
+            self.from_ts,
+            self.to_ts,
+            self.resolution,
+            aggregation_methods=self.aggregation_methods,
         )
         if self.row_filter:
             data = pandas_filter_rows(data, self.row_filter)
 
         x_tag_names = [tag.name for tag in self.tag_list]
         y_tag_names = [tag.name for tag in self.target_tag_list]
 
+        # If we have multiple aggregation methods then the column-names will be e.g
+        # "tag 1_max", not "tag 1", so we must do some stuff to pick out the right
```

Comment:
It most definitely does. Or rather, the user must filter on `tag 1_max` instead of `tag 1`.

Reply:
Maybe add to one of the (many) locations of examples? I guess we have a separate repo this example folder and the gordo test project. IMO the [gordo-test-project config](https://github.com/equinor/gordo-test-project/blob/master/kubernetes-deployment/components/config/config.yaml) would be good since that's what we used to "validate" bigger changes and is doing a good job at doubling as an example resource.